src/web/mjs/connectors/Delitoon.mjs

```diff
@@ -35,13 +35,13 @@ export default class Delitoon extends Connector {
         let data = await this.fetchDOM(request, 'div.genres-page div.content ul li');
         return data.filter(element => element.querySelector('h3'))
             .map(element => {
-                let title = element.querySelector('div.container-infos h3').textContent.trim();
-                let link = element.querySelector('li a');
+                let title = element.querySelector('div.genres-page div.content ul li div.container-infos h3').textContent.trim();
+                let link = element.closest('a');
```

Comment:
The search direction has been changed, but the chapter query is still the same, was this intentional?
`querySelectors()` searches the DOM tree downwards (children)
`closest()` searches the DOM tree upwards (parents)

Reply:
Already reverted with successive commit